### PR TITLE
Adjust the prompt at release to remind releasing querydsl-contrib

### PR DESCRIPTION
**Pull request by Shredder121**
I don't think a backport is necessary for this one.
One reminder sounds okay I think.

### CODE REVIEW

**Comment by johnktims** [comment]
Ha! Elegant solution, sir. ;)

**Comment by timowest** [comment]
Maybe in release.sh instead? At only for major versions? ;)

File: release.sh [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+echo -e "\x1b[49m\x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[36m&\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[97m \x1b[39m"
+echo -e "\x1b[49m\x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[36m&\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[39m"
+echo -e "\x1b[49m\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[36m&\x1b[97m \x1b[39m"
+echo -e "\x1b[49m\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[36m&\x1b[97m \x1b[97m \x1b[37m:\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[37m*\x1b[97m \x1b[97m \x1b[97m \x1b[97m \x1b[39m"
```
Comment (Shredder121) [inline release.sh]: Just mentioning: this is not serious, but take a look anyway. :wink:

**Comment by johnktims** [comment]
How about a reminder to update the soft link to the latest documentation?

**Comment by Shredder121** [comment]
Good point!
Shall I also just repush with `[ci skip]`?
It doesn't affect the build.

**Comment by johnktims** [comment]
Yes.

**Comment by Shredder121** [comment]
I made adjustments, comments?

**Comment by timowest** [comment]
Looks good, shall I merge?

**Comment by Shredder121** [comment]
Yes, that's fine with me.

**Comment by johnktims** [comment]
Nice work, Ruben.

**Pull request merged**